output/containers.eslint.txt

```diff
@@ -1,12 +1,15 @@
 
-> @talend/react-containers@0.155.1 lint:es /home/travis/build/Talend/ui/packages/containers
+> @talend/react-containers@0.156.0 lint:es /home/travis/build/Talend/ui/packages/containers
 > eslint --config ../../.eslintrc src
 
 The react/require-extension rule is deprecated. Please use the import/extensions rule from eslint-plugin-import instead.
 
 /home/travis/build/Talend/ui/packages/containers/src/ActionSplitDropdown/ActionSplitDropdown.test.js
   4:10  error  'ActionSplitDropdown' is defined but never used  no-unused-vars
 
+/home/travis/build/Talend/ui/packages/containers/src/DeleteResource/deleteResource.sagas.js
+  106:4  warning  Unexpected console statement  no-console
```

Comment:
Please remove that console log

Reply:
No never mind, it's from another PR, don't bother with that, it's fixed in another PR too